app/managers.py

```diff
@@ -0,0 +1,46 @@
+import sqlite3
+from models import Actor
```

Comment:
The import statement for `Actor` should be corrected to `from app.models import Actor` to match the module structure.